Dockerfile

```diff
@@ -23,6 +23,8 @@ RUN apk update && apk add --update libev gmp git
 RUN chmod -R 755 /var
 
 COPY --from=build /home/opam/_build/default/src/ocamlorg_web/bin/main.exe /bin/server
+COPY --from=build /home/opam/_build/default/asset _build/default/asset
+COPY --from=build /home/opam/_build/default/data/media _build/default/data/media
```

Comment:
I need to test whether this is correct. I was thinking, I should probably try to `docker build` and `docker run` the docker container. However, this is what I get:

```
$ sudo docker build .
Sending build context to Docker daemon  52.88MB
Step 1/22 : FROM ocaml/opam:alpine-3.17-ocaml-4.14 as build
 ---> d6b3acb10d6c
Step 2/22 : RUN sudo apk update && sudo apk add --update libev-dev openssl-dev gmp-dev oniguruma-dev
 ---> Running in 07a17d876c02
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/x86_64/APKINDEX.tar.gz
ERROR: https://dl-cdn.alpinelinux.org/alpine/v3.17/main: temporary error (try again later)
fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/x86_64/APKINDEX.tar.gz
ERROR: https://dl-cdn.alpinelinux.org/alpine/v3.17/community: temporary error (try again later)
fetch https://dl-cdn.alpinelinux.org/alpine/edge/main/x86_64/APKINDEX.tar.gz
ERROR: https://dl-cdn.alpinelinux.org/alpine/edge/main: temporary error (try again later)
fetch https://dl-cdn.alpinelinux.org/alpine/edge/community/x86_64/APKINDEX.tar.gz
ERROR: https://dl-cdn.alpinelinux.org/alpine/edge/community: temporary error (try again later)
fetch https://dl-cdn.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz
ERROR: https://dl-cdn.alpinelinux.org/alpine/edge/testing: temporary error (try again later)
v3.17.2-24-g9bb746ae153 [https://dl-cdn.alpinelinux.org/alpine/v3.17/main]
v3.17.2-35-g33db3ffa533 [https://dl-cdn.alpinelinux.org/alpine/v3.17/community]
v20230208-795-g69e308aa808 [https://dl-cdn.alpinelinux.org/alpine/edge/main]
v20230208-798-ge0d946eda73 [https://dl-cdn.alpinelinux.org/alpine/edge/community]
v20230208-799-gb4c6cbb034d [https://dl-cdn.alpinelinux.org/alpine/edge/testing]
5 errors; 40350 distinct packages available
```

Reply:
Ok my bad, updating the Docker version fixed that :laughing: